packages/plugins/rubygem-foreman_rescue/rubygem-foreman_rescue.spec

```diff
@@ -0,0 +1,89 @@
+# Generated from foreman_rescue-1.0.0.gem by gem2rpm -*- rpm-spec -*-
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_rescue
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 1.0.0
+Release: 1%{?dist}
+Summary: Provides the ability to boot a host into a rescue system
+Group: Development/Languages
+License: GPL-3.0
+URL: https://github.com/dm-drogeriemarkt/foreman_rescue
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+# end generated dependencies
+
+%description
+Foreman Plugin to provide the ability to boot a host into a rescue system.
+
+
+%package doc
+Summary: Documentation for %{pkg_name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{pkg_name}.
+
+%prep
+%{?scl:scl enable %{scl} - << \EOF}
+gem unpack %{SOURCE0}
+%{?scl:EOF}
+
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+%{?scl:scl enable %{scl} - << \EOF}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:EOF}
+
+%build
+# Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
+gem build %{gem_name}.gemspec
+%{?scl:EOF}
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install
+%{?scl:EOF}
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -pa .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+%files
+%dir %{gem_instdir}
+%license %{gem_instdir}/LICENSE
+%{gem_instdir}/app
+%{gem_instdir}/config
+%{gem_instdir}/db
+%{gem_libdir}
+%{gem_instdir}/locale
+%exclude %{gem_cache}
+%{gem_spec}
+
+%files doc
+%doc %{gem_docdir}
+%doc %{gem_instdir}/README.md
+%{gem_instdir}/Rakefile
+%{gem_instdir}/test
+
+%changelog
+* Tue Jun 05 2018 Dirk Goetz <dirk.goetz@netways.de> 1.0.0-1
+- Add rubygem-foreman_rescue generated by gem2rpm using the scl template
```

Comment:
Should this be using the plugin template?

Reply:
Note I didn't check this for the other 2 packages but they're probably also actual plugins rather than a library.

Reply:
Yes, I was focused on it being scl instead of nonscl so I did not recognize the foreman_plugin is scl, too.